Adding KashCorpSA

Murdabenne opened the pull request:
Shared Assets (virtual mod) for KAP and RoversAndRoadsters mods.

Dazpoet commented:
- File extension is .netkan
- You need to remove the comma on line 11
- What is the reasoning behind the hardcoded `"version"` field rather than using the scraped one?

Murdabenne commented:
scraped version differs depending on which one of the 2 sources you look at - this is shared between the 2 follow-on mods Im adding afterward.  So I decided to version the psud-mod instead, to be able to point it at whichver is more up to date as the base.  I rechecked the build with netkan on my Win8.1 box and it accepted it with the extra comma, no error reported.  Perhaps a newer version of netkan is needed?   I have deleted the comma, renamed the file, and will redo the pull.

Dazpoet commented:
netkan.exe will accept a lot of things, such as .txt files but the buildbot runs it through a json validator. You can do your own validation at e.g. www.jsonlint.com to see if your file looks ok.

Rather than opening multiple PRs you can instead push changes to this one :)
